test/e2e/scenario/logs.scenario.ts

```diff
@@ -84,4 +84,15 @@ describe('logs', () => {
       expect(unreachableRequest.http.status_code).toEqual(0)
       expect(unreachableRequest.error!.stack).toContain('TypeError')
     })
+
+  createTest('returns user configuration')
```

Comment:
nice intention, however I'd say that for this kind of update unit test should be enough.
e2e tests (wdio) are more time expensive so we tend to only add some when unit/integration tests (karma) are not enough